server/routes/auth.js

```diff
@@ -1,5 +1,40 @@
 const jwt = require('express-jwt')
 const jwksRsa = require('jwks-rsa')
+const request = require('superagent')
+
+/*
+* this object holds the required information to acquire a new access token for the purpose of retrieving
+* the user's role using machine-to-machine.
+* The clientid and secret are for the API Explorer Application
+* NOTE: the access token passed to routes from the client would not work, because it has a different audience.
+*/
+const apiExplorer = {
```

Comment:
This doesn't look like the most appropriate name. Is `authProvider` (or something else) more appropriate?